pubsub/Makefile

```diff
@@ -0,0 +1,5 @@
+gen_query_parser:
```

Comment:
Don't think we need it here (at least for now). the `gen_query_parser` command will be used very rarely and the whole Makefile is just one command right now.